src/models/UrlShorten.js

```diff
@@ -1,11 +1,12 @@
 const mongoose = require("mongoose");
 const { Schema } = mongoose;
 const urlShortenSchema = new Schema({
-    long_url: String,
+    long_url: { type: String, required: true },
     clipped_url: String,
-    urlCode: String,
-    created_by: String,
+    urlCode: String ,
```

Comment:
Great job... What does urlCode represent? If it is the id of the url, i do not think it needs to be in the schema as Mongoose will create one automatically as "id"

Reply:
yes like an id 
and the new short url will be like 

trims/Urlcode